test/powershell/Modules/Microsoft.PowerShell.Security/SecureString.Tests.ps1

```diff
@@ -41,4 +41,18 @@ Describe "SecureString conversion tests" -Tags "CI" {
         $ss2 = $encodedStr | ConvertTo-SecureString -Key $key
         $ss2 | ConvertFrom-SecureString -AsPlainText | Should -BeExactly $testString
     }
+
+    It "Using invalid secure string with ConvertFrom-SecureString produces an exception message without value" {
+        { ConvertFrom-SecureString "1234" } |
+            Should -Throw `
+                -ErrorId "CannotConvertArgumentNoMessage,Microsoft.PowerShell.Commands.ConvertFromSecureStringCommand" `
+                -ExpectedMessage 'Cannot bind parameter ''SecureString''. Cannot convert the value of type "System.String" to type "System.Security.SecureString".'
```

Comment:
We never use error messages in tests because of localization. I suggest positive test - explicitly check that a message hasn't sensitive value.